ext/opcache/jit/zend_jit.c

```diff
@@ -150,7 +150,7 @@ static int zend_jit_assign_to_variable(dasm_State    **Dst,
                                        zend_jit_addr   var_addr,
                                        uint32_t        var_info,
                                        uint32_t        var_def_info,
-                                       zend_uchar      val_type,
+                                       uint8_t      val_type,
```

Comment:
```suggestion
                                       uint8_t         val_type,
```

Reply:
and PR title should be changed as well ;-)

Reply:
The commit message is what matters, and I had updated it already when I switched to `uint8_t`. GH copies the commit message to the PR only on the initial submission, and never updates it again. Keep both up to date would be extra work, and since the PR text will never be part of the git repository, I figured it's time not well invested.

Reply:
I understand your POV, but I do not agree. PR title should be kept in sync for documentation purposes and also because someone may merge this PR via GH GUI, which will use the GH title.